lua/cfc_notifications/client/cl_dnotificationbutton.lua

```diff
@@ -25,11 +27,18 @@ end
 function PANEL:Think()
```

Comment:
This seems like a heavy Think function. You sure nothing can be moved out of here?